src/jvm/main/org/jetbrains/kotlinx/lincheck/strategy/managed/ManagedStrategy.kt

```diff
@@ -1312,6 +1531,20 @@ abstract class ManagedStrategy(
         }
     }
 
+    /**
+     * Current method context in a call stack.
+     */
+    sealed interface CallContext {
+        /**
+         * Indicates that current method is static.
+         */
+        data object StaticCallContext: CallContext
```

Comment:
Why `data`?

Reply:
IDEA highlights this because it's a pattern (object in a sealed hierarchy)
![image](https://github.com/JetBrains/lincheck/assets/45040429/9974ee27-cb08-4f6c-b889-825ef9d22d07)

Reply:
Also, data objects have better toString() implementations, which is useful for debugging